src/types/publication_types/messages.types.ts

```diff
@@ -1,16 +1,25 @@
 import { Registration } from '../registration.types';
 
-export interface MessageItem {
+export enum MessageType {
+  DoiRequest = 'DoiRequest',
+  Support = 'Support',
+}
+
+export interface Message {
   date: string;
   id: string;
   identifier: string;
-  isDoiRequestRelated: boolean;
   owner: string;
   sender: string;
   text: string;
 }
 
-export interface Message {
-  messages: MessageItem[];
+export interface MessageCollection {
+  messageType: MessageType;
+  messages: Message[];
+}
+
+export interface SupportRequest {
```

Comment:
blir det egentlig riktig å kalle denne typen `SupportRequest`? Det høres ut som den typen en bruker når en skal gjøre et kall til et `Support`-endepunkt. Blir det ikke mer rett å kalle det `SupportResponse`? Eller tenker du at denne typen er en blanding av Support og DoiRequest ?

Reply:
Tenker at dette er blanding av Support og DoiRequest ja (og Approval når vi etterhvert får støtte for det. En samlebetegnelse for alt som har meldinger og kan dukke opp på Mine Meldinger/Arbeidsliste